tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/compose/HelmUpgradeExtensionIT.java

```diff
@@ -1,128 +1,80 @@
-    private static final @NotNull HelmChartContainer HELM_CHART_CONTAINER =
-            new HelmChartContainer().withNetwork(NETWORK);
+    @Container
     private static final @NotNull HiveMQContainer HIVEMQ_CONTAINER =
             new HiveMQContainer(HIVEMQ_DOCKER_IMAGE).withLogLevel(Level.DEBUG)
-                    .withNetwork(NETWORK)
+                    .withNetwork(network)
                     .withNetworkAliases("remote");
-    @SuppressWarnings("NotNullFieldNotInitialized")
-    private static @NotNull KubernetesClient client;
-
-    @BeforeAll
-    @Timeout(value = 5, unit = TimeUnit.MINUTES)
-    static void baseSetup() {
-        HIVEMQ_CONTAINER.start();
-        HELM_CHART_CONTAINER.start();
-        client = HELM_CHART_CONTAINER.getKubernetesClient();
-    }
-
-    @AfterAll
-    @Timeout(value = 5, unit = TimeUnit.MINUTES)
-    static void baseTearDown() {
-        HELM_CHART_CONTAINER.stop();
-        HIVEMQ_CONTAINER.stop();
-        NETWORK.close();
-    }
-
-    @BeforeEach
-    @Timeout(value = 5, unit = TimeUnit.MINUTES)
-    void setup() throws Exception {
-        HELM_CHART_CONTAINER.createNamespace(NAMESPACE);
-        HELM_CHART_CONTAINER.installOperatorChart(OPERATOR_RELEASE_NAME);
-    }
-
-    @AfterEach
-    @Timeout(value = 5, unit = TimeUnit.MINUTES)
-    void tearDown() throws Exception {
-        HELM_CHART_CONTAINER.uninstallRelease(PLATFORM_RELEASE_NAME, NAMESPACE, true);
-        HELM_CHART_CONTAINER.uninstallRelease(OPERATOR_RELEASE_NAME, "default");
-    }
 
     @Test
     @Timeout(value = 5, unit = TimeUnit.MINUTES)
-    void withBridgeConfiguration_enableDisableBridged() throws Exception {
-        final var customResourceName = "test-hivemq-platform";
-
+    void withBridgeConfiguration_enableDisableBridge() throws Exception {
         final var hivemqContainerNetwork =
                 HIVEMQ_CONTAINER.getContainerInfo().getNetworkSettings().getNetworks().values().stream().findFirst();
         assertThat(hivemqContainerNetwork).isPresent();
 
         // setup bridge configuration
-        final URL resource = getClass().getResource("/bridge-config.xml");
-        final var configMapData = Files.readString(Path.of(Objects.requireNonNull(resource).toURI()));
-        final var bridgeConfigMap = new ConfigMapBuilder().withNewMetadata()
-                .withName("test-bridge-configuration")
-                .endMetadata()
-                .withData(Map.of("config.xml",
-                        configMapData.replace("<host>remote</host>",
-                                "<host>" + hivemqContainerNetwork.get().getIpAddress() + "</host>")))
-                .build();
-        client.configMaps().inNamespace(NAMESPACE).resource(bridgeConfigMap).create();
+        final var bridgeConfiguration = readResourceFile("bridge-config.xml").replace("<host>remote</host>",
+                "<host>" + hivemqContainerNetwork.get().getIpAddress() + "</host>");
```

Comment:
Same here, does this really work or should we simply replace this with `localhost`?